README.md

```diff
@@ -7,7 +7,7 @@
 
 Production environment:
 
-- https://api.hel.fi/kultus-beta/graphql
+- https://palvelutarjotin-api.prod.kuva.hel.ninja/graphql
```

Comment:
Should we already mention the new Platta address also?

Reply:
Added Platta addresses and status of the use of the KuVa and Platta addresses on 2023-02-24 to README and force pushed the [changes](https://github.com/City-of-Helsinki/palvelutarjotin/compare/bd3adb622ff4aabe50ca6c060629e28d4ecff15f..b129018ac993c1a6de58e4f49ba68e11f9f58ebc).